kotlin-redux/src/main/kotlin/redux/Imports.kt

```diff
@@ -0,0 +1,9 @@
+@file:JsModule("redux")
+package redux
+
+@JsName("createStore")
+internal external class Store<out S>(reducer: (S, dynamic) -> S, preloadedState: S) {
```

Comment:
Can it be this interface instead of `dynamic`?

```
{
  val type: String
  val action: RAction
}
```

Reply:
`dispatch` should keep accepting `dynamic` (or maybe `Any`) though, because there are middleware

Reply:
It works for the most part, but the original dispatch() function expects a plain JavaScript object. Passing a kotlin object that extends an interface breaks it.

Reply:
What about `reducer` signature?